modules/chemical_reactions/include/userobjects/ThermochimicaData.h

```diff
@@ -55,9 +61,16 @@ class ThermochimicaNodalData : public NodalUserObject
     std::vector<double> _mol_fraction;
   };
 
-  const Data & getNodalData(dof_id_type node_id) const;
+  const Data & getNodalData(dof_id_type id) const;
+  const Data & getElementData(dof_id_type id) const;
+
+  // universal data access
+  const Data & getData(dof_id_type id) const;
 
 protected:
+  // get current node or element ID
+  auto currentID();
```

Comment:
const?

Reply:
could be, but that function gets removed in #25970 anyways